### ISSUE TITLE
Fade between images

### ISSUE THREAD

**Issue by colejd** [sindresorhus/Plash#47]
When the image changes, it's a pretty jarring snap between the two. It'd be nice to have fading between them as an option.

**Comment by sindresorhus** [comment]
By "image", I assume you mean "website". And yes, this is planned. We need to change it so that Plash loads the new website in a new hidden web view, and if it succeeds, we fade it in over the existing web view, and then close the existing web view. That would create a seamless experience.

**Comment by colejd** [comment]
This appears to be a duplicate of #9.

**Comment by sindresorhus** [comment]
@colejd Not really. #9 is about fading in the initial load of the web view. That's very easy. Just need to animate it after detecting it's done loading. This issue is about fading when navigating, which means we need two make two web views and fade between them. This issue could potentially surpass #9, but #9 would be easier to do in the short-term.
